src/api/app/models/kiwi/image.rb

```diff
@@ -103,6 +125,10 @@ def outdated?
 
     package.kiwi_image_outdated?
   end
+
+  def packages
+    package_groups.where(kind: 'image', pattern_type: 'onlyRequired').map(&:packages).flatten
```

Comment:
Here is the documentation:
https://doc.opensuse.org/projects/kiwi/doc/

Not sure if it is really necessary to distinguish between the pattern / type etc for now. 
Maybe we can a scope?